idf_component.yml

```diff
@@ -0,0 +1,2 @@
+description: "Arduino Wiring-based Framework for the Espressif ESP32, ESP32-S and ESP32-C series of SoCs"
+url: "https://github.com/espressif/arduino-esp32"
```

Comment:
@magicarm22 Please add targets:  ESP32, ESP32-S2 and ESP32-C3
and maybe some tags: like `arduino`